frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt

```diff
@@ -60,35 +83,10 @@ class DeleteBranchOnSlideOutSuggestionDialog(project: Project) :
               .component
               .apply {
             mnemonic = KeyEvent.VK_R
-            isEnabled = true
             isSelected = false
-          }(pushX)
-          cell {
-            button(
-                    getString(
-                        "action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.delete-text")) {
-                  delete = true
-                  close(OK_EXIT_CODE)
-                }
-                .component
-                .apply { mnemonic = KeyEvent.VK_D }
-            button(
-                    getString(
-                        "action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.keep-text")) {
-                  delete = false
-                  close(OK_EXIT_CODE)
-                }
-                .component
-                .apply { mnemonic = KeyEvent.VK_K }
-            button(
-                    getString(
-                        "action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.cancel-text")) {
-                  delete = true
-                  close(CANCEL_EXIT_CODE)
-                }
-                .component
-                .apply { mnemonic = KeyEvent.VK_C }
           }
         }
       }
+
+  override fun createSouthPanel(): JComponent? = null
```

Comment:
Hmm maybe drop a comment in code why this is needed (something needs to be hidden this way?)

Reply:
commented (amended)